contents/blog/interview-snapshot-guide.md

```diff
@@ -0,0 +1,82 @@
+---
+date: 2023-03-08
+title: "How to turn user interviews into actionable interview snapshots"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - annika-schmid
+featuredImage: ../images/blog/posthog-blog-at-desk.png
+featuredImageType: full
+category: Inside PostHog
+tags:
+  - Guides
+  - Product engineers
+---
+
+Regularly talking to users is an important habit for any product team, but it's wasted without an efficient way to share what you learn. At PostHog, we favor interview snapshots. Championed by Teresa Torres in the book [Continuous Discovery Habits](https://www.amazon.co.uk/Continuous-Discovery-Habits-Discover-Products/dp/1736633309), we like snapshots because:
```

Comment:
```suggestion
Regularly talking to users is an important habit for any product team, but it's wasted without an efficient way to share what you learn. At PostHog, we favor interview snapshots. Championed by Teresa Torres in the book [Continuous Discovery Habits](https://www.amazon.com/Continuous-Discovery-Habits-Discover-Products/dp/1736633309), we like snapshots because:
```